spec/controllers/idv/confirmations_controller_spec.rb

```diff
@@ -24,39 +26,67 @@
       init_idv_session
     end
 
-    describe 'all questions answered correctly' do
-      it 'shows success' do
+    context 'all answers correct' do
```

Comment:
Can you put `complete_idv_session(true)` in a before block?

My understanding is that when you use `context` you should always try to have a before block, even if you only have one spec, just to separate the context setup from the assertion/test itself.